### PR TITLE
feat: rarity

### PR DESCRIPTION
**Pull request by topether21**
Enable send an utxo

image

image

### CODE REVIEW

File: src/assets/scss/elements/_uninscribed_stats.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.uninscribed-sats .container {
+  border-radius: 8px;
+  overflow: hidden;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
```
[the remark this answers is not in the document]
Comment (topether21) [reply src/assets/scss/elements/_uninscribed_stats.scss:4]: image
? @andredevjs

**Review by habibitcoin — APPROVED** [review]
I think for usability, we need to be able to see the UTXO value somehow
image
Not blocking tho, will log as follow up

Overall, amazing work!! Fix merge conflicts, and lets ship!

**Review by habibitcoin — CHANGES_REQUESTED** [review]
1 small bug, looks like https://deploy-preview-303.test.deezy.place/inscription/8a8ed539eeba089a64f95a16fb031724474ec4326ae8009bf4ec703d87a94673i0 view bids button isnt working

**Review by habibitcoin — APPROVED** [review]
ready to go!

**Pull request merged**